examples/common/python/utility/file_utils.py

```diff
@@ -55,7 +60,31 @@ def read_json_file(input_file, data_dir = ['./', '../', '/']) :
     """
     Utility function to read a JSON file
     """
-    file_name = find_file_in_path(input_file, data_dir)
+    file_name = find_file_in_paths(input_file, data_dir)
     with open(file_name, "r") as input_json_file :
-        input_json = input_json_file.read().rstrip('\n')
-    return input_json
+        input_lines = input_json_file.read().splitlines()
+    flattened_json = ""
+    for line in input_lines :
+        flattened_json += line
+    return flattened_json
+
+#---------------------------------------------------------------------------------------------
+def write_result_data_to_json_file(file_name, input_data, data_dir ='./') :
```

Comment:
1. You have introduced space before ":" above. There is an inconsistency in the code with this. The preferred way is not to introduce space before ":". This inconsistency is not because of your code. It is already there.
2. PEP-8 says, don't introduce space around = for function arguments with default values. So, it should be data_dir='./'

Reply:
Updated the 2nd one. Left the first one as is. We can integrate the PEP8 tool to catch such issues and carry out a comprehensive cleanup in that case as a single task.